README.md

```diff
@@ -58,71 +58,6 @@ It is also possible to use both a configuration file and cli arguments.  In thes
 
 Finally, the location of the config file can be changed either through the cli argument `--config FILE, -c FILE`, or the environment variable `$RKE2_CONFIG_FILE`.
 
-## RPM Repositories
-
-Signed RPMs are published for RKE2 within the `rpm-testing.rancher.io` and `rpm.rancher.io` RPM repositories. If you run the https://get.rke2.io script on nodes supporting RPMs, it will use these RPM rpeos by default. But you can also install them yourself.
-
-The RPMs provide `systemd` units for managing `rke2`, but will need to be configured via configuration file before starting the services for the first time.
-
-### Enterprise Linux 7
-
-In order to use the RPM repository, on a CentOS 7 or RHEL 7 system, run the following bash snippet:
-
-```bash
-cat << EOF > /etc/yum.repos.d/rancher-rke2-1-18-latest.repo
-[rancher-rke2-common-latest]
-name=Rancher RKE2 Common Latest
-baseurl=https://rpm.rancher.io/rke2/latest/common/centos/7/noarch
-enabled=1
-gpgcheck=1
-gpgkey=https://rpm.rancher.io/public.key
-
-[rancher-rke2-1-18-latest]
-name=Rancher RKE2 1.18 Latest
-baseurl=https://rpm.rancher.io/rke2/latest/1.18/centos/7/x86_64
-enabled=1
-gpgcheck=1
-gpgkey=https://rpm.rancher.io/public.key
-EOF
-```
-
-### Enterprise Linux 8
-
-In order to use the RPM repository, on a CentOS 8 or RHEL 8 system, run the following bash snippet:
-
-```bash
-cat << EOF > /etc/yum.repos.d/rancher-rke2-1-18-latest.repo
-[rancher-rke2-common-latest]
-name=Rancher RKE2 Common Latest
-baseurl=https://rpm.rancher.io/rke2/latest/common/centos/8/noarch
-enabled=1
-gpgcheck=1
-gpgkey=https://rpm.rancher.io/public.key
-
-[rancher-rke2-1-18-latest]
-name=Rancher RKE2 1.18 Latest
-baseurl=https://rpm.rancher.io/rke2/latest/1.18/centos/8/x86_64
-enabled=1
-gpgcheck=1
-gpgkey=https://rpm.rancher.io/public.key
-EOF
-```
-
-### Installing
-
-After this, you can either run:
-
-```bash
-yum -y install rke2-server
-```
-or
-
-```bash
-yum -y install rke2-agent
-```
-
-The RPM will install a corresponding `rke2-server.service` or `rke2-agent.service` systemd unit that can be invoked like: `systemctl start rke2-server`. Make sure that you configure `rke2` before you start it, by following the `Configuration File` instructions below.
-
```

Comment:
Removing all of this without any linkage to the new doc seems unfortunate.

Reply:
Updated.